pkg/hub/managedclusterset/controller.go

```diff
@@ -161,35 +165,62 @@ func (c *managedClusterSetController) syncClusterSet(ctx context.Context, cluste
 		emptyCondition.Reason = "ClustersSelected"
 		emptyCondition.Message = fmt.Sprintf("%d ManagedClusters selected", count)
 	}
-	meta.SetStatusCondition(&clusterSetCopy.Status.Conditions, emptyCondition)
+	meta.SetStatusCondition(&clusterSet.Status.Conditions, emptyCondition)
 
 	// skip update if cluster set status does not change
-	if reflect.DeepEqual(clusterSetCopy.Status.Conditions, clusterSet.Status.Conditions) {
+	if reflect.DeepEqual(clusterSet.Status.Conditions, originalClusterSet.Status.Conditions) {
 		return nil
 	}
 
-	_, err = c.clusterClient.ClusterV1beta1().ManagedClusterSets().UpdateStatus(ctx, clusterSetCopy, metav1.UpdateOptions{})
+	_, err = c.clusterClient.ClusterV1beta1().ManagedClusterSets().UpdateStatus(ctx, clusterSet, metav1.UpdateOptions{})
 	if err != nil {
-		return fmt.Errorf("failed to update status of ManagedClusterSet %q: %w", clusterSetCopy.Name, err)
+		return fmt.Errorf("failed to update status of ManagedClusterSet %q: %w", clusterSet.Name, err)
 	}
 
 	return nil
 }
 
-func getClusterSet(obj interface{}) string {
-	accessor, err := meta.Accessor(obj)
+func (c *managedClusterSetController) enqueueClusterClusterSet(cluster *v1.ManagedCluster) {
+	clusterSets, err := clusterv1beta1.GetClusterSetsOfCluster(cluster, c.clusterSetLister)
 	if err != nil {
-		utilruntime.HandleError(fmt.Errorf("error to get accessor of object: %v", obj))
-		return ""
+		utilruntime.HandleError(fmt.Errorf("error to get GetClusterSetsOfCluster. Error %v", err))
+		return
 	}
-	labels := accessor.GetLabels()
-	if len(labels) == 0 {
-		return ""
+	for _, clusterSet := range clusterSets {
+		c.queue.Add(clusterSet.Name)
 	}
+}
+func (c *managedClusterSetController) enqueueUpdateClusterClusterSet(oldCluster, newCluster *v1.ManagedCluster) {
```

Comment:
one more blank line. 

Also please add some comment i this func.

Reply:
done